cli/.cs.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should this file be named something like `cs-dev.json`?  I'm concerned about it being picked up unintentionally.

Reply:
What is this file's intended use?

Reply:
It's sole purpose is as a reasonable default config for developers testing the CLI

Reply:
Yeah seems fine.